seed/src/event.rs

```diff
@@ -72,7 +88,7 @@ impl Event {
                 name: proj.name().to_owned(),
                 description: proj.description().to_owned(),
             },
-            provider,
+            User::guess_from_peer(api, provider).await?,
```

Comment:
So here we've tracked the `Project` and we have a `provider`. So we can actually do:
```rust
storage.get_rad_self_of(urn, provider)
```
in the above and return `proj` and the `user` for the `proivder`